### DISCUSSION TITLE
Show container build output during ddev start?

### DISCUSSION THREAD

**Discussion by deviantintegral** [ddev/ddev#4786] [Q&A]
Is there a way we can get docker-compose build output to show while a container is building? While it shows if there's an error, it can be confusing if a container simply has a larger set of additions on top of it, or if something in the build process is hanging like a bad network connection.

Locally, I work around this when developing by running `ddev debug compose-config > .ddev/.ddev-docker-compose-full.yaml` and then `~/.ddev/bin/docker-compose -f .ddev/.ddev-docker-compose-full.yaml build web --progress=plain`, but that doesn't help in the "something is unexpectedly slow" situation, or CI where tailing verbose logs live is more helpful.

**Comment by rfay** [comment]
I would like this also. I have to do as you did from time to time. 

`ddev debug refresh` currently does this stuff, but I'd like it if it would do a build stage separately with full output. `ddev debug refresh -v` for example. 

https://github.com/ddev/ddev/blob/bbb2caca961784aa68d0511848823464a009f14b/cmd/ddev/cmd/debug-refresh.go#L42-L54

Thanks for exploring this territory; a PR would be welcome, but I imagine we'll get to it at some point.

It's also possible that `ddev start` could have a 2-stage start with `build` and then `up`, and capture the build stuff with `-v` or something.

**Reply by deviantintegral** [reply]
A `ddev build` would be really good. That way, I could be building containers while leaving existing ones running. As well, it would be one step closer to being able to push up customized web container images to a registry to save on installation time.
